deps/exokit-bindings/magicleap/src/magicleap.cc

```diff
@@ -630,7 +739,52 @@ void MLContext::LifecycleInit() {
   lifecycle_callbacks.on_pause = onPause;
   lifecycle_callbacks.on_resume = onResume;
 
-  lifecycle_status = MLLifecycleInit(&lifecycle_callbacks, (void*)&application_context);
+#if 1
+  {
+    std::ofstream filestr;
+    std::ofstream errstr;
+    //std::cout << "Redirecting stdout\n";
+    std::streambuf* oldsb = redirect_output("mlsdk_log1.txt", filestr, std::cout);
```

Comment:
Any way to not force a file write?

Reply:
cout and cerr can be redirected to std::ostringstream, but I'm not sure about stdout and stderr. It might require file handles.